functions/composer-storage-trigger/test/index.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2018 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const proxyquire = require(`proxyquire`).noCallThru();
+const sinon = require(`sinon`);
+const test = require(`ava`);
+
+function getSample () {
+  const bodyJson = {};
+  const body = {
+    json: sinon.stub().resolves(bodyJson)
+  };
+  const FetchMock = sinon.stub().resolves(body);
+
+  return {
+    program: proxyquire(`../`, {
+      'node-fetch': FetchMock
+    }),
+    mocks: {
+      fetch: FetchMock,
+      body: body,
+      bodyJson: bodyJson
+    }
+  };
+}
+
+test.cb.serial(`Does nothing for deleted files`, (t) => {
+  const event = {
+    data: {
+      file: `some-file`
+    }
+  };
+  const expectedMsg = `Something bad happened.`;
+  const sample = getSample();
+  sample.mocks.bodyJson.error = expectedMsg;
+
+  sample.program.triggerDag(event, (err, message) => {
```

Comment:
Should we switch to promises at some point? (In a separate PR, of course.)

Reply:
I didn't realize we could return promises with GCF. Yeah, that'd be ideal.